File: Lab 2/Sum.c
#include <stdio.h>

int main (void)
{
  // int value1, value2, sum;
  float value1, value2, sum;
  value1 = 50;
  value2 = 25;
  sum = value1 + value2;
  // printf("The sum of %d and %d is %d\n", value1, value2, sum);
  // printf("The sum of %f and %f is %f\n", value1, value2, sum);
  printf("The sum of %.1f and %f is %f\n", value1, value2, sum);
  return 0;
}

File: Lab 2/Sum.lua
-- "Any variable name is assumed to be global unless explicitly declared as a
-- local." (§3.2)
local value1, value2 = 50.0, 25.0
local sum = value1 + value2

-- "The format string follows the same rules as the ISO C function sprintf."
--   https://www.lua.org/manual/5.3/manual.html#pdf-string.format
print(string.format("The sum of %.1f and %f is %f", value1, value2, sum))
